apps/examples/wc-angular12/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "wc-angular12",
+    "version": "0.0.0",
+    "scripts": {
+        "ng": "ng",
+        "dev": "ng serve",
+        "build": "ng build",
+        "watch": "ng build --watch --configuration development"
+    },
+    "private": true,
+    "dependencies": {
+        "@angular/animations": "15.2.0",
+        "@angular/common": "15.2.0",
+        "@angular/compiler": "15.2.0",
+        "@angular/core": "15.2.0",
+        "@angular/forms": "15.2.0",
+        "@angular/platform-browser": "15.2.0",
+        "@angular/platform-browser-dynamic": "15.2.0",
+        "@angular/router": "15.2.0",
+        "@justeattakeaway/pie-button": "0.10.2",
+        "@webcomponents/webcomponentsjs": "2.8.0",
+        "rxjs": "~7.8.0",
+        "tslib": "2.3.0",
+        "zone.js": "~0.12.0"
+    },
+    "devDependencies": {
+        "@angular-devkit/build-angular": "15.2.6",
+        "@angular/cli": "~15.2.6",
+        "@angular/compiler-cli": "15.2.0",
+        "@types/jasmine": "~4.3.0",
+        "jasmine-core": "~4.5.0",
+        "karma": "~6.4.0",
+        "karma-chrome-launcher": "~3.1.0",
+        "karma-coverage": "~2.2.0",
+        "karma-jasmine": "~5.1.0",
+        "karma-jasmine-html-reporter": "~2.0.0",
+        "typescript": "~4.9.4"
```

Comment:
Can you lock these dependencies (remove any `~) and from the dependencies above too.

Also, do I'm guessing we don't need the `package-lock.json` (as there's a `yarn.lock`)